tests/unit/cli_test_extensions.py

```diff
@@ -449,3 +450,36 @@ def remove_ws(inputs, join=False):
     if join:
         ret_lines = ''.join(ret_lines)
     return ret_lines
+
+
+def setup_mock_connection(mock_items, namespace):
+    """
+    Return FakedWBEMConnection with the specified mock items in the specified
+    namespace.
+
+    Parameters:
+
+      mock_items (iterable of mock items): List of mock items to be added
+        to the mock environment.
+
+        A mock item can be:
+        * string: path name of MOF file to be compiled
+        * string: MOF string to be compiled
+        * list/tuple: List of CIM objects to be added
```

Comment:
Would it make sense to include a python file as one of the mock_item types.

Reply:
I looked at that but it is not easy, so I left it out for now. Can be extened in the future.
NO CHANGE